pipenv.el

```diff
@@ -384,13 +387,26 @@ to latest compatible versions."
 ;; Integration with 3rd party packages.
 ;;
 
+(defun pipenv-verify-python-checkers ()
```

Comment:
@inpefess as `pipenv-verify-python-checkers` is a "private" function (and it is not interactive) it probably should follow the convention for private functions in Emacs lisp, so `pipenv--verify-python-checkers`. What do you think?

Reply:
Yes, you're right. I'm pretty new to Emacs Lisp and didn't know about this convention yet. Thanks.